homework-submission/react-hyf/react-hyf-client/src/React_HW3/index.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+class HomeworkWeek3 extends React.Component {
+  constructor() {
+    super();
+    this.state = {};
```

Comment:
Initialing the state with no properties will be working perfectly in all case, but for a best practice and in case working in a strict mode applications, you don't have to initialize the state only, but with it's properties.

Reply:
```suggestion
   class HomeworkWeek3 extends React.Component {
  constructor(props) {
    super(props);
    this.state = {};
  }
```

Reply:
Will it be ok like that